webpack/InsightsHostDetailsTab/InsightsTotalRiskChartWrapper.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import InsightsTotalRiskChart from './InsightsTotalRiskChart';
+import { isNotRhelHost } from '../ForemanRhCloudHelpers';
+
+export const InsightsTotalRiskChartWrapper = props => {
+  if (props.status === 'RESOLVED') {
+    return (
+      !isNotRhelHost(props.hostDetails) && <InsightsTotalRiskChart {...props} /> // check for RHEL hosts
+    );
+  }
+  return null;
```

Comment:
Is it better to return an empty element here? `</>`?

Reply:
Unsure, the [docs](https://legacy.reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering) recommend `null`, but I read somewhere that there is a case for returning `false` instead...